pkg/cloudclient/aws/private.go

```diff
@@ -404,8 +411,8 @@ func (c *Client) setCloudImage(cloudImageID string) (string, error) {
 // - create instance and wait till it gets ready, wait for userdata script execution
 // - find unreachable endpoints & parse output, then terminate instance
 // - return `c.output` which stores the execution results
-func (c *Client) validateEgress(ctx context.Context, vpcSubnetID, cloudImageID string, kmsKeyID string, timeout time.Duration, p proxy.ProxyConfig) *output.Output {
-	c.WriteDebugLogs(ctx, fmt.Sprintf("Using configured timeout of %s for each egress request", timeout.String()))
```

Comment:
I like the renaming of "vpcSubnetId" to "subnetId" and same for "amiId", since it makes sense in aws context. One question: why the preference for `c.logger.Debug` as opposed to the existing `c.WriteDebugLogs` ? It seems that `WriteDebugLogs` also adds it to the output struct. Do we not want that here?

Reply:
WriteDebugLogs is the way to go...I must've made a mistake rebasing, thanks for catching this!